### PR TITLE
Hide PGP link in Profil when in app

### PR DESCRIPTION
**Pull request by patrickvenetz**

### CODE REVIEW

File: components/Profile/Contact.js [hunk 1]
```diff
@@ -202,7 +204,7 @@ const Contact = ({ user, isEditing, onChange, values, errors, dirty, t }) => {
           />
         )}
       </div>
-      {user.pgpPublicKeyId &&
+      {!inNativeIOSApp && user.pgpPublicKeyId &&
```
Comment (tpreusse) [inline components/Profile/Contact.js]: did you test that it doesn't cause the same issue on Android?

Comment (patrickvenetz) [reply components/Profile/Contact.js]: No, I did not will. Do so shortly.

Comment (patrickvenetz) [reply components/Profile/Contact.js]: Checked on two mobile Android devices, neither of them is acting on that link.

Comment (tpreusse) [reply components/Profile/Contact.js]: IMO the right approach would then be to either hide it in on both platforms or remove the link on both plattform and still show the key id (still—somehwat—useful info).

**Pull request merged**